### PR TITLE
Edit account name

### PR DESCRIPTION
**Pull request by djhahe**
### Summary (Please recap what you changed or fixed)
Add button to edit account name

### Checklist (If you won't pass this checklist please comment why)

-   [ ] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

**Comment by gaupoit** [comment]
hi @djhahe , please create the related ticket for this PR.

**Review by gaupoit — APPROVED** [review]

File: client/src/components/web-extension/Common/Account/AccountManagerModal.jsx [hunk 1]
```diff
@@ -6,25 +6,32 @@ import Button from 'react-bootstrap/Button';
...
 import { useNavigate } from 'react-router-dom';
 import EnterPasswordModal from '@cd/web-extension/Common/LoginModal/EnterPasswordModal';
+import { updateAccountName } from '@cd/hooks/useServiceWorker';
 
 export const AccountManagerModal = ({ isOpen, onClose, isUserExisting, ...restProps }) => {
 	const dispatch = useDispatch();
 	const selectedWallet = useSelector(getSelectedWallet);
 	const navigate = useNavigate();
 	const [showEnterPassword, setShowEnterPassword] = useState(false);
+	const [editingAccount, setEditingAccount] = useState();
```
Comment (hoailinh1210) [inline client/src/components/web-extension/Common/Account/AccountManagerModal.jsx:33]: Should default is `false` for the clear default value

File: client/src/components/web-extension/Common/Account/AccountManagerModal.jsx [hunk 2]
```diff
@@ -6,25 +6,32 @@ import Button from 'react-bootstrap/Button';
...
+	const [newAccountName, setNewAccountName] = useState();
```
Comment (hoailinh1210) [inline client/src/components/web-extension/Common/Account/AccountManagerModal.jsx:34]: Should have default value

**Review by duc-q — APPROVED** [review]
Apart from minor changes from @hoailinh1210 , this looks good to me.

**Pull request merged**